.gitignore

```diff
@@ -1,12 +1,15 @@
-/Build/
-.DS_Store
-*_extdep/
-*.pyc
-__pycache__/
-tags/
-.vscode/
-*.bak
-BuildConfig.conf
-
-# Ignore unused UDK PKGs.
-/Conf/
+/Build/
+.DS_Store
+*_extdep/
+*.pyc
+__pycache__/
+tags/
+.vscode/
+*.bak
+BuildConfig.conf
+
+# Submodules
+MU_BASECORE/
+
+# Ignore unused UDK PKGs.
+/Conf/

```

Comment:
Did the whole file change because of an accidental line ending change?

Reply:
yeah, line ending change from LF to CRLF

Reply:
Gotcha, purposeful line ending change :)